modules/vocabularies/src/main/frontend/src/vocabularies-admin-page.jsx

```diff
@@ -17,10 +17,27 @@
 //  under the License.
 //
 
+import { Grid, Typography } from "@material-ui/core";
 import React from "react";
 
 function VocabulariesAdminPage(props) {
-  return(<div></div>);
+  return (
+    <Grid spacing={5} container direction="column" justify="space-between">
```

Comment:
The order of the attributes should be from more significant to less important, so that someone reading the code gets to know what this is sooner. In this case, the most important thing is that this is a container, with its items put in a column.

```suggestion
    <Grid container direction="column" spacing={5} justify="space-between">
```

And I'm not sure the `justify` attribute is needed at all. Since this is all a single column, with flexible height, all its items will just be placed one under the other.

```suggestion
    <Grid container direction="column" spacing={5}>
```